addons/crm/__manifest__.py

```diff
@@ -10,6 +10,7 @@
     'website': 'https://www.odoo.com/app/crm',
     'depends': [
         'base_setup',
+        'base_install_request',
```

Comment:
I think that this need an upgrade script to install the module if not already installed 🧐